### PR TITLE
change to gulpfile so codeship will work

### PR DESCRIPTION
**Pull request by andianderson522**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -23,6 +23,17 @@ gulp.task('test', ['todo', 'lint'], function performTests() {
     .pipe(mocha({'reporter': 'spec', 'timeout': 100}));
 });
 
+gulp.task('test2', ['lint'], function performTests() {
```
Comment (kniels21) [inline gulpfile.js]: Im curious. Would adding the `process.exit` calls to the original test target have screwed up running gulp locally?

Comment (andianderson522) [inline gulpfile.js]: then the watch wont work, it would exit the first run through tests.

Comment (kniels21) [inline gulpfile.js]: Ah ok. Then perhaps a comment explaining why this second test task is here. Other than that looks fine.

**Comment by ghvaldez** [comment]
looks good to me

**Pull request merged**